src/lld/backend_patterns/repository/template/Readme.md

```diff
@@ -0,0 +1,81 @@
+# Repository and dao pattern
+1. Rest controller
+   1. Responsible for rest api request and deserialize request
+   2. Create/ Inject respective service class
+   3. Handle exception from service and convert to respective response status code
+2. Service
+   1. Responsible for managing repository
```

Comment:
as well as caching repository requests. Some of HLD aspects like service to service communication, business logic implementation